src/chttpd/src/chttpd_misc.erl

```diff
@@ -106,6 +106,11 @@ handle_utils_dir_req(Req, _) ->
     send_method_not_allowed(Req, "GET,HEAD").
 
 handle_all_dbs_req(#httpd{method='GET'}=Req) ->
+    handle_all_dbs_or_dbs_info_req_helper(Req);
```

Comment:
We could probably shorten it to `handle_all_dbs_info_req` without losing too much context.